api/client/commands.go

```diff
@@ -1611,7 +1611,7 @@ func (cli *DockerCli) CmdPs(args ...string) error {
 	)
 	cmd.Require(flag.Exact, 0)
 
-	cmd.Var(&flFilter, []string{"f", "-filter"}, "Filter output based on conditions provided")
+	cmd.Var(&flFilter, []string{"f", "-filter"}, "Filter output based on conditions provided.  Valid filters:\nid=<id> - containers with id of <id>\nname=<name> - containers with name of <name>\nexited=<int> - containers with exit code of <int>\nstatus=(restarting|running|paused|exited)")
```

Comment:
no, just update the current commits.  You may also want to check, I'm pretty sure there are other spots in the docs where the additional text describing the current filters would be good - like the man pages but @moxiegirl would know for sure.